farm.pde

```diff
@@ -2,4 +2,26 @@ class Farm extends Building {
   Farm(Cell initialLocation) {
     super(initialLocation, "Farm");
   }
+
+  void draw() {
+    super.draw();
+    fill(255, 255, 255);
+    textSize(4);
+    text("F", this.pos.x, this.pos.y);
+  }
 }
+
+class Crop extends Building {
```

Comment:
Love this idea. Does this mean if more than one `Farmer` is assigned to the same `Farm` they'll plant more `Crop` tiles?

Reply:
Yep. It's not currently in there, but I also plan on making it so that if any crop buildings are unmanned (maybe you got rid of their farmer), a new farmer will assign themself to that crop.